sources/packages/api/src/services/cra-income-verification/cra-income-verification.service.ts

```diff
@@ -249,4 +249,32 @@ export class CRAIncomeVerificationService extends RecordDataModelService<CRAInco
       }
     }
   }
+
+  /**
+   * Get the student and supporting users (if any) income verification details for an applications.
+   * @param applicationId application id.
+   * @returns income verification details for an applications.
+   */
+  async allIncomeVerificationsForAnApplication(
+    applicationId: number,
```

Comment:
Looking at it as an independent service, should we consider including studentId in the where?